config/audit.php

```diff
@@ -36,7 +36,7 @@
 
     'user' => [
         'morph_prefix' => 'user',
-        'guards'       => [
+        'guards' => [
```

Comment:
Lets leave the array alignment alone, please.

Reply:
fixed